app/src/main/java/org/fossasia/susi/ai/data/contract/IUtilModel.kt

```diff
@@ -0,0 +1,24 @@
+package org.fossasia.susi.ai.data.contract
+
+import org.fossasia.susi.ai.rest.responses.susi.LoginResponse
+import retrofit2.Response
+
+/**
+ * Interface for Util Model
+ *
+ * Created by chiragw15 on 10/7/17.
+ */
+interface IUtilModel {
```

Comment:
No. It is "utility" model i.e common for every feature. Just like a util class which can be used from anywhere. It can be used from any presenter. It basically is used to work on context based utility functions like getting strings from resources, getting preferences etc.